homeassistant/components/alexa/__init__.py

```diff
@@ -45,12 +56,42 @@
     "https://api.fe.amazonalexa.com/v3/events",
 ]
 
+# RANGE_CONTROLLER_SCHEMA = vol.Schema(
+#    {
+#        vol.Required(CONF_NAME): cv.string,
+#        vol.Required(CONF_ASSET): vol.In(AlexaGlobalCatalog.__dict__.values()),
+#       More work to do here
+#    }
+# )
+
+MODE_CONTROLLER_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): cv.string,
+        vol.Required(CONF_ASSET): vol.In(AlexaGlobalCatalog.__dict__.values()),
+        vol.Required(CONF_ATTRIBUTE): cv.string,
+        vol.Required(CONF_VALUES): {
+            vol.Required(cv.string): {
+                vol.In(AlexaModeController.supported_locales): [cv.string],
+            },
+        },
```

Comment:
We do not allow customizing the YAML of this internal core integration.
You can create a smart skill: https://www.home-assistant.io/integrations/alexa.smart_home/#create-an-amazon-alexa-smart-home-skill.

But the base principle for the Alexa Smart Home integration is to be a proxy for Home Assistant entities without the need for users to change any config. The base language in code is en-us. The UI allows translations, so we communicate in English to Alexa.

Reply:
Ok then i will delelte the PR